style.css

```diff
@@ -113,3 +113,146 @@ header {
   list-style-type: none;
   padding-bottom: 5rem;
 }
+
+.projects {
+  margin: 20% 6% 10% 6%;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+}
```

Comment:
- [ ] Please I highly recommend using CSS flexbox for displaying individual `cards` per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). 😄

Reply:
added flex column